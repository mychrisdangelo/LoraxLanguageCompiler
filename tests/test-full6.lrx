int main()
{
	char a;
	char b;
	char c;
	char d;
	char e;
	char f;
	string s;

	s = "hello, world\n";

	a = '\t';
	b = 'b';
	c = '\n';
	d = 'd';
	e = '\"';
	f = '\\';

	print(a, b, c, d, e, f, '\n');
	print(s);
}
