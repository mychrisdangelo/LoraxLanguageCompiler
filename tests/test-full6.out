	b
d\
hello, world
bye
